essencium-backend/src/testIntegration/java/de/frachtwerk/essencium/backend/test/integration/controller/UserControllerIntegrationTest.java

```diff
@@ -725,6 +725,69 @@ void testFilterUsersByName() throws Exception {
         .andExpect(jsonPath("$.content.[0].id", is(user2.getId()), Long.class));
   }
 
+  @Test
+  void testFilterUsersByEmail() throws Exception {
```

Comment:
For a better readability: you can nested your tests with `@Nested` and a human readable name with `@DisplayName`.

Furthermore, we should only test one case per test method